setup.py

```diff
@@ -185,12 +185,13 @@ def run(self):
     "simpleitk": [],
     "spe": [],
     "swf": [],
-    "tifffile": ["tifffile"],
+    "tifffile": ["tifffile", "imagecodecs"],
```

Comment:
This seems an unrelated change. Can you explain why you added it?

Reply:
`imagecodecs` breaks on pypy, so I am removing it again.